en/config_rover/mecanum.md

```diff
@@ -0,0 +1,298 @@
+   The proportional gain is multiplied with this error and that value is added to the motor command.
+   This way disturbances like uneven grounds or external forces can be compensated.
+
+   ::: tip
+   This parameter can be tuned the same way as [RM_MAX_THR_YAW_R](#RM_YAW_RATE_P_TUNING).
+   If you tuned [RM_MAX_THR_YAW_R](#RM_MAX_YAW_RATE) well, you might only need a very small value.
+   :::
+
+5. (Optional) [RM_YAW_RATE_I](#RM_YAW_RATE_I) [-]: Integral gain of the closed loop yaw controller.
+   The integral gain accumulates the error between the desired and actual yaw rate over time and that value is added to the motor command.
+
+   ::: tip
+   The integrator gain is usually not necessary for the yaw rate setpoint as this is usually a fast changing value.
+   Leave this parameter at zero unless necessary, as it can have negative side effects such as overshooting or oscillating around the setpoint.
+   :::
+
+The rover is now ready to drive in [Acro mode](../flight_modes_rover/mecanum.md#acro-mode).
+
+## Stabilized Mode
+
+::: warning
+For this mode to work properly [Acro mode](#acro-mode) must've already been configured!
+:::
+
+For [Stabilized mode](../flight_modes_rover/mecanum.md#stabilized-mode) the controller utilizes a closed loop yaw controller, which creates a yaw rate setpoint to control the yaw when it is active:
+
+![Cascaded PID for yaw control](../../assets/airframes/rover/rover_differential/cascaded_pid_for_yaw.png)
+
+Unlike the closed loop yaw rate, this controller has no feed-forward term.
+Therefore you only need to tune the closed loop gains:
+
+1. [RM_YAW_P](#RM_YAW_P) [-]: Proportional gain for the closed loop yaw controller.
+
+   ::: tip
+   In stabilized mode the closed loop yaw control is only active when driving a straight line (no yaw rate input).
+   To tune it set [RM_YAW_I](#RM_YAW_I) to zero and start with a value of 1 for [RM_YAW_P](#RM_YAW_P).
+   Put the rover into stabilized mode and move the left stick of your controller up and/or down to drive forwards/backwards.
+   Disarm the rover and from the flight log plot the _yaw_setpoint_ from the [RoverMecanumSetpoint](../msg_docs/RoverMecanumSetpoint.md) message and the _actual_yaw_ from the [RoverMecanumStatus](../msg_docs/RoverMecanumStatus.md) message over each other.
+   Increase/Decrease the parameter until you are satisfied with the setpoint tracking.
+   :::
+
+2. [RM_YAW_I](#RM_YAW_I) [-]: Integral gain for the closed loop yaw controller.
+
+   ::: tip
+   For the closed loop yaw control an integrator gain is useful because this setpoint is often constant for a while and an integrator eliminates steady state errors that can cause the rover to never reach the setpoint.
+   In [Auto Modes](#auto-modes) there will be a further elaboration on why an integrator is necessary for the yaw controller.
```

Comment:
How do we tune this one - you explain the other

Reply:
Updated this in https://github.com/PX4/PX4-user_guide/pull/3411/commits/ab30f0666db49e0a694bfe92f2f89ca2f9760945